generator/src/routes/+page.svelte

```diff
@@ -152,6 +152,9 @@
   function onChangeSignOption(value: string) {
     clearError()
     signOption = value
+    if (signOption === SignOption.UNSIGNED) {
+      isValid = false
+    }
```

Comment:
Not sure why this is here? Unsigned uses a different API so the value of this doesn't matter, and this just changes the value to the wrong default.

Reply:
Since apparently the code isn't doing that yet, I created another task to change the API endpoint in that case.

Reply:
```suggestion
```